plugins/catalog-creator/src/components/CatalogForm/CatalogForm.tsx

```diff
@@ -276,18 +283,69 @@ export const CatalogForm = ({
                   <div>
                     <FieldHeader
                       fieldName="Owner"
-                      tooltipText="A reference to the owner (commonly a team), that bears ultimate responsibility for the component, and has the authority and capability to develop and maintain it"
+                      tooltipText="A reference to the owner (commonly a team), that bears ultimate responsibility for the entity, and has the authority and capability to develop and maintain it"
                       required
                     />
                     <Controller
                       name={`entities.${index}.owner`}
                       control={control}
                       render={({ field: { onChange, onBlur, value } }) => (
-                        <CatalogSearch
-                          onChange={onChange}
+                        <Autocomplete
+                          value={
+                            value
+                              ? (fetchOwners?.value?.find(
+                                  ownerEntity =>
+                                    ownerEntity.metadata.name === value,
+                                ) ?? null)
+                              : null
+                          }
                           onBlur={onBlur}
-                          value={value}
-                          entityList={fetchOwners.value || []}
+                          onChange={(_, newValue) => {
+                            const names = newValue?.metadata?.name ?? '';
+                            onChange(names);
+                          }}
+                          options={fetchOwners.value || []}
+                          getOptionLabel={option => {
+                            return option.metadata.name;
+                          }}
+                          filterOptions={(options, state) => {
+                            const inputValue = state.inputValue.toLowerCase();
+                            return options.filter(option => {
+                              const name = option.metadata.name.toLowerCase();
+                              const title = (
+                                option.metadata.title ?? ''
+                              ).toLowerCase();
+                              return (
+                                name.includes(inputValue) ||
+                                title.includes(inputValue)
+                              );
+                            });
```

Comment:
Denne var litt lang, kunne vi flyttet ut funksjonaliteten og kalt på den her i stedet? 
```suggestion
                            const inputValue = state.inputValue.toLowerCase();
                            return options.filter(option => {
                              const name = option.metadata.name.toLowerCase();
                              const title = (
                                option.metadata.title ?? ''
                              ).toLowerCase();
                              return (
                                name.includes(inputValue) ||
                                title.includes(inputValue)
                              );
                            });
```